app/templates/rentals/index.hbs

```diff
@@ -0,0 +1,7 @@
+<ul class="results">
+  {{#each model as |rentalUnit|}}
+    {{#link-to "rentals.show" rentalUnit.slug}}
```

Comment:
I agree we shouldn't lose that functionality.  One of us should do some research on if there's a way to supress/override the link to behavior with the image expansion.